apiserver/facades/client/bundle/bundle.go

```diff
@@ -96,3 +101,20 @@ func (b *bundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 	}
 	return results, nil
 }
+
+// ExportBundle exports the current model configuration as bundle.
+func (b *BundleAPI) ExportBundle() (params.StringResult, error) {
+	model, err := b.backend.Export()
```

Comment:
There needs to be a permission check that the user has read access to the model.